src/cli/test_command.zig

```diff
@@ -861,6 +861,12 @@ pub const TestCommand = struct {
             files: []const PathString,
             allocator: std.mem.Allocator,
             pub fn begin(this: *@This()) void {
+                if (this.vm.hot_reload == .watch) {
```

Comment:
This isn't the right place for it

Here is the right place:

https://github.com/oven-sh/bun/blob/efe987e8d12e824dde840b56cbb704feabe26ed1/src/bun.zig#L1292

We should pass `true` to `clear_terminal`.